android/assets/jsons/Civ V - Vanilla/Beliefs.json

```diff
@@ -241,6 +241,56 @@
         "name": "World Church",
         "type": "Founder",
         "uniques": ["[+1 Culture] for every [5] global followers [in foreign cities]"]
+    },
+
+    ////////////////////////////////////// Enhancer beliefs ///////////////////////////////////////
+    
+    {
+        "name": "Defender of the Faith",
+        "type": "Enhancer",
+        "uniques": ["[+20]% Strength for [Non-City] units in [Friendly Land]"],
```

Comment:
Cities can't utilize this unique anyway since they aren't units
So All units

Reply:
We do have uniques containing text like "[+10]% Strength for [City] units", so I felt that really this unique was an addition to clearly distinguish between those cases

Reply:
Perhaps, but "non-city units" feels really off. Cities aren't units.
The Param itself could be useful in other places, I agree, but not here